src/components/LayoutHeader/Nav/LinkItems/index.tsx

```diff
@@ -1,33 +1,43 @@
-import React, { useRef, useState } from 'react'
+import React from 'react'
 import cn from 'classnames'
 
 import Link from 'gatsby-theme-iterative-docs/src/components/Link'
-import { OtherToolsPopup, CommunityPopup } from '../Popup'
+import {
+  OtherToolsPopup,
+  CommunityPopup,
+  OtherPopup,
+  IPopupProps
+} from '../Popup'
 
 import { ReactComponent as ArrowUpSVG } from '../../../../../static/img/arrow-up-icon.svg'
 import { ReactComponent as ArrowDownSVG } from '../../../../../static/img/arrow-down-icon.svg'
 
 import { logEvent } from 'gatsby-theme-iterative-docs/src/utils/front/plausible'
 import { getFirstPage } from 'gatsby-theme-iterative-docs/src/utils/shared/sidebar'
+import usePopup, { IUsePopupReturn } from '../../../../gatsby/hooks/usePopup'
 
 const docsPage = getFirstPage()
 
 import * as styles from './styles.module.css'
 
-type PopupName = 'CommunityPopup' | 'OtherToolsPopup'
+type PopupName = 'communityPopup' | 'otherToolsPopup' | 'otherPopup'
 
 interface INavLinkData {
   href: string
   eventType: string
   text: string
+  className?: string
 }
 
 interface INavLinkPopupData {
   text: string
-  popup: PopupName
+  popupName: PopupName
+  Popup: React.FC<IPopupProps>
+  className?: string
+  href?: string
 }
 
-const navLinkItemsData: Array<INavLinkData | INavLinkPopupData> = [
+export const navLinkItemsData: Array<INavLinkData | INavLinkPopupData> = [
```

Comment:
I wasn't sure where "Learn" should go in the nav. For now, I placed it after "Support" :)